### ISSUE TITLE
Blockhound 1.0.2 doesn't work with OpenJDK 14

### ISSUE THREAD

**Issue by juergenzimmermann** [reactor/BlockHound#101]
I'm getting tons of `IllegalArgumentException`s with message _Unsupported class file major version 58_ as soon as I use `BlockHound.install()` in the beginning of `main()`. An extract of a few exceptions is attached at the end. Using _OpenJDK 14_ without `BlockHound.install()` works fine for my Kotlin-based Spring app which is compiled to JVM target 13 (being the latest supported target of the Kotlin compiler).

## Expected Behavior
It should support the latest OpenJDK release.

## Actual Behavior
Tons of exceptions.

## Steps to Reproduce
Download OpenJDK 14 https://jdk.java.net/14 and use it instead of e.g. OpenJDK 11.

## Your Environment
OpenJDK 14, Windows 10 Enterprise

* Reactor version(s) used: Dysprosium-SR5
* Other relevant libraries versions (eg. `netty`, ...):
* JVM version (`javar -version`): openjdk 14 2020-03-17
* OS and version (eg `uname -a`): Windows 10.0.17134

Extract of the exceptions:

```
...
[Byte Buddy] ERROR java.net.SocketImpl [null, module java.base, loaded=false]
java.lang.IllegalArgumentException: Unsupported class file major version 58
        at reactor.blockhound.shaded.net.bytebuddy.jar.asm.ClassReader.<init>(ClassReader.java:195)
        at reactor.blockhound.shaded.net.bytebuddy.jar.asm.ClassReader.<init>(ClassReader.java:176)
        at reactor.blockhound.shaded.net.bytebuddy.jar.asm.ClassReader.<init>(ClassReader.java:162)
        at reactor.blockhound.shaded.net.bytebuddy.utility.OpenedClassReader.of(OpenedClassReader.java:82)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$Default.parse(TypePool.java:681)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$Default.doDescribe(TypePool.java:667)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$AbstractBase.describe(TypePool.java:408)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$AbstractBase$Hierarchical.describe(TypePool.java:484)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$DescriptionStrategy$Default$3.apply(AgentBuilder.java:3582)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.resolve(AgentBuilder.java:10524)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.doTransform(AgentBuilder.java:10494)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.transform(AgentBuilder.java:10457)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.access$1500(AgentBuilder.java:10223)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer$Java9CapableVmDispatcher.run(AgentBuilder.java:10907)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer$Java9CapableVmDispatcher.run(AgentBuilder.java:10845)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.transform(AgentBuilder.java:10413)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer$ByteBuddy$ModuleSupport.transform(Unknown Source)
        at java.instrument/sun.instrument.TransformerManager.transform(TransformerManager.java:188)
        at java.instrument/sun.instrument.InstrumentationImpl.transform(InstrumentationImpl.java:563)
        at java.base/sun.nio.ch.ServerSocketAdaptor.<init>(ServerSocketAdaptor.java:74)
        at java.base/sun.nio.ch.ServerSocketAdaptor.lambda$create$0(ServerSocketAdaptor.java:65)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
        at java.base/sun.nio.ch.ServerSocketAdaptor.create(ServerSocketAdaptor.java:67)
        at java.base/sun.nio.ch.ServerSocketChannelImpl.socket(ServerSocketChannelImpl.java:128)
        at java.base/sun.nio.ch.PipeImpl$Initializer$LoopbackConnector.run(PipeImpl.java:121)
        at java.base/sun.nio.ch.PipeImpl$Initializer.run(PipeImpl.java:76)
        at java.base/sun.nio.ch.PipeImpl$Initializer.run(PipeImpl.java:61)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
        at java.base/sun.nio.ch.PipeImpl.<init>(PipeImpl.java:171)
        at java.base/sun.nio.ch.SelectorProviderImpl.openPipe(SelectorProviderImpl.java:50)
        at java.base/java.nio.channels.Pipe.open(Pipe.java:155)
        at java.base/sun.nio.ch.WindowsSelectorImpl.<init>(WindowsSelectorImpl.java:142)
        at java.base/sun.nio.ch.WindowsSelectorProvider.openSelector(WindowsSelectorProvider.java:44)
        at java.base/java.nio.channels.Selector.open(Selector.java:295)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:160)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:214)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:227)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:231)
        at org.apache.kafka.clients.admin.KafkaAdminClient.createInternal(KafkaAdminClient.java:427)
        at org.apache.kafka.clients.admin.Admin.create(Admin.java:69)
        at org.apache.kafka.clients.admin.AdminClient.create(AdminClient.java:49)
        at org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner.createAdminClient(KafkaTopicProvisioner.java:239)
        at org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner.doProvisionConsumerDestination(KafkaTopicProvisioner.java:209)
        at org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner.provisionConsumerDestination(KafkaTopicProvisioner.java:181)
        at org.springframework.cloud.stream.binder.kafka.streams.KafkaStreamsBinderUtils.prepareConsumerBinding(KafkaStreamsBinderUtils.java:84)
        at org.springframework.cloud.stream.binder.kafka.streams.KStreamBinder.doBindConsumer(KStreamBinder.java:102)
        at org.springframework.cloud.stream.binder.kafka.streams.KStreamBinder.doBindConsumer(KStreamBinder.java:52)
        at org.springframework.cloud.stream.binder.AbstractBinder.bindConsumer(AbstractBinder.java:143)
        at org.springframework.cloud.stream.binding.BindingService.doBindConsumer(BindingService.java:169)
        at org.springframework.cloud.stream.binding.BindingService.bindConsumer(BindingService.java:115)
        at org.springframework.cloud.stream.binding.AbstractBindableProxyFactory.createAndBindInputs(AbstractBindableProxyFactory.java:112)
        at org.springframework.cloud.stream.binding.InputBindingLifecycle.doStartWithBindable(InputBindingLifecycle.java:58)
        at java.base/java.util.LinkedHashMap$LinkedValues.forEach(LinkedHashMap.java:647)
        at org.springframework.cloud.stream.binding.AbstractBindingLifecycle.start(AbstractBindingLifecycle.java:57)
        at org.springframework.cloud.stream.binding.InputBindingLifecycle.start(InputBindingLifecycle.java:34)
        at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:182)
        at org.springframework.context.support.DefaultLifecycleProcessor.access$200(DefaultLifecycleProcessor.java:53)
        at org.springframework.context.support.DefaultLifecycleProcessor$LifecycleGroup.start(DefaultLifecycleProcessor.java:360)
        at org.springframework.context.support.DefaultLifecycleProcessor.startBeans(DefaultLifecycleProcessor.java:158)
        at org.springframework.context.support.DefaultLifecycleProcessor.onRefresh(DefaultLifecycleProcessor.java:122)
        at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:894)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:553)
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:758)
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:750)
        at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
        at com.acme.receiver.ApplicationKt.main(Application.kt:88)
[Byte Buddy] ERROR java.net.SocketOptions [null, module java.base, loaded=false]
java.lang.IllegalArgumentException: Unsupported class file major version 58
        at reactor.blockhound.shaded.net.bytebuddy.jar.asm.ClassReader.<init>(ClassReader.java:195)
        at reactor.blockhound.shaded.net.bytebuddy.jar.asm.ClassReader.<init>(ClassReader.java:176)
        at reactor.blockhound.shaded.net.bytebuddy.jar.asm.ClassReader.<init>(ClassReader.java:162)
        at reactor.blockhound.shaded.net.bytebuddy.utility.OpenedClassReader.of(OpenedClassReader.java:82)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$Default.parse(TypePool.java:681)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$Default.doDescribe(TypePool.java:667)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$AbstractBase.describe(TypePool.java:408)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$AbstractBase$Hierarchical.describe(TypePool.java:484)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$DescriptionStrategy$Default$3.apply(AgentBuilder.java:3582)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.resolve(AgentBuilder.java:10524)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.doTransform(AgentBuilder.java:10494)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.transform(AgentBuilder.java:10457)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.access$1500(AgentBuilder.java:10223)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer$Java9CapableVmDispatcher.run(AgentBuilder.java:10907)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer$Java9CapableVmDispatcher.run(AgentBuilder.java:10845)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.transform(AgentBuilder.java:10413)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer$ByteBuddy$ModuleSupport.transform(Unknown Source)
        at java.instrument/sun.instrument.TransformerManager.transform(TransformerManager.java:188)
        at java.instrument/sun.instrument.InstrumentationImpl.transform(InstrumentationImpl.java:563)
        at java.base/sun.nio.ch.ServerSocketAdaptor.<init>(ServerSocketAdaptor.java:74)
        at java.base/sun.nio.ch.ServerSocketAdaptor.lambda$create$0(ServerSocketAdaptor.java:65)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
        at java.base/sun.nio.ch.ServerSocketAdaptor.create(ServerSocketAdaptor.java:67)
        at java.base/sun.nio.ch.ServerSocketChannelImpl.socket(ServerSocketChannelImpl.java:128)
        at java.base/sun.nio.ch.PipeImpl$Initializer$LoopbackConnector.run(PipeImpl.java:121)
        at java.base/sun.nio.ch.PipeImpl$Initializer.run(PipeImpl.java:76)
        at java.base/sun.nio.ch.PipeImpl$Initializer.run(PipeImpl.java:61)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
        at java.base/sun.nio.ch.PipeImpl.<init>(PipeImpl.java:171)
        at java.base/sun.nio.ch.SelectorProviderImpl.openPipe(SelectorProviderImpl.java:50)
        at java.base/java.nio.channels.Pipe.open(Pipe.java:155)
        at java.base/sun.nio.ch.WindowsSelectorImpl.<init>(WindowsSelectorImpl.java:142)
        at java.base/sun.nio.ch.WindowsSelectorProvider.openSelector(WindowsSelectorProvider.java:44)
        at java.base/java.nio.channels.Selector.open(Selector.java:295)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:160)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:214)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:227)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:231)
        at org.apache.kafka.clients.admin.KafkaAdminClient.createInternal(KafkaAdminClient.java:427)
        at org.apache.kafka.clients.admin.Admin.create(Admin.java:69)
        at org.apache.kafka.clients.admin.AdminClient.create(AdminClient.java:49)
        at org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner.createAdminClient(KafkaTopicProvisioner.java:239)
        at org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner.doProvisionConsumerDestination(KafkaTopicProvisioner.java:209)
        at org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner.provisionConsumerDestination(KafkaTopicProvisioner.java:181)
        at org.springframework.cloud.stream.binder.kafka.streams.KafkaStreamsBinderUtils.prepareConsumerBinding(KafkaStreamsBinderUtils.java:84)
        at org.springframework.cloud.stream.binder.kafka.streams.KStreamBinder.doBindConsumer(KStreamBinder.java:102)
        at org.springframework.cloud.stream.binder.kafka.streams.KStreamBinder.doBindConsumer(KStreamBinder.java:52)
        at org.springframework.cloud.stream.binder.AbstractBinder.bindConsumer(AbstractBinder.java:143)
        at org.springframework.cloud.stream.binding.BindingService.doBindConsumer(BindingService.java:169)
        at org.springframework.cloud.stream.binding.BindingService.bindConsumer(BindingService.java:115)
        at org.springframework.cloud.stream.binding.AbstractBindableProxyFactory.createAndBindInputs(AbstractBindableProxyFactory.java:112)
        at org.springframework.cloud.stream.binding.InputBindingLifecycle.doStartWithBindable(InputBindingLifecycle.java:58)
        at java.base/java.util.LinkedHashMap$LinkedValues.forEach(LinkedHashMap.java:647)
        at org.springframework.cloud.stream.binding.AbstractBindingLifecycle.start(AbstractBindingLifecycle.java:57)
        at org.springframework.cloud.stream.binding.InputBindingLifecycle.start(InputBindingLifecycle.java:34)
        at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:182)
        at org.springframework.context.support.DefaultLifecycleProcessor.access$200(DefaultLifecycleProcessor.java:53)
        at org.springframework.context.support.DefaultLifecycleProcessor$LifecycleGroup.start(DefaultLifecycleProcessor.java:360)
        at org.springframework.context.support.DefaultLifecycleProcessor.startBeans(DefaultLifecycleProcessor.java:158)
        at org.springframework.context.support.DefaultLifecycleProcessor.onRefresh(DefaultLifecycleProcessor.java:122)
        at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:894)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:553)
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:758)
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:750)
        at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
        at com.acme.receiver.ApplicationKt.main(Application.kt:88)
[Byte Buddy] ERROR sun.net.NetProperties [null, module java.base, loaded=false]
java.lang.IllegalArgumentException: Unsupported class file major version 58
        at reactor.blockhound.shaded.net.bytebuddy.jar.asm.ClassReader.<init>(ClassReader.java:195)
        at reactor.blockhound.shaded.net.bytebuddy.jar.asm.ClassReader.<init>(ClassReader.java:176)
        at reactor.blockhound.shaded.net.bytebuddy.jar.asm.ClassReader.<init>(ClassReader.java:162)
        at reactor.blockhound.shaded.net.bytebuddy.utility.OpenedClassReader.of(OpenedClassReader.java:82)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$Default.parse(TypePool.java:681)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$Default.doDescribe(TypePool.java:667)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$AbstractBase.describe(TypePool.java:408)
        at reactor.blockhound.shaded.net.bytebuddy.pool.TypePool$AbstractBase$Hierarchical.describe(TypePool.java:484)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$DescriptionStrategy$Default$3.apply(AgentBuilder.java:3582)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.resolve(AgentBuilder.java:10524)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.doTransform(AgentBuilder.java:10494)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.transform(AgentBuilder.java:10457)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.access$1500(AgentBuilder.java:10223)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer$Java9CapableVmDispatcher.run(AgentBuilder.java:10907)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer$Java9CapableVmDispatcher.run(AgentBuilder.java:10845)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer.transform(AgentBuilder.java:10413)
        at reactor.blockhound.shaded.net.bytebuddy.agent.builder.AgentBuilder$Default$ExecutingTransformer$ByteBuddy$ModuleSupport.transform(Unknown Source)
        at java.instrument/sun.instrument.TransformerManager.transform(TransformerManager.java:188)
        at java.instrument/sun.instrument.InstrumentationImpl.transform(InstrumentationImpl.java:563)
        at java.base/java.net.SocketImpl.lambda$usePlainSocketImpl$0(SocketImpl.java:70)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:312)
        at java.base/java.net.SocketImpl.usePlainSocketImpl(SocketImpl.java:71)
        at java.base/java.net.SocketImpl.<clinit>(SocketImpl.java:67)
        at java.base/sun.nio.ch.ServerSocketAdaptor.<init>(ServerSocketAdaptor.java:74)
        at java.base/sun.nio.ch.ServerSocketAdaptor.lambda$create$0(ServerSocketAdaptor.java:65)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
        at java.base/sun.nio.ch.ServerSocketAdaptor.create(ServerSocketAdaptor.java:67)
        at java.base/sun.nio.ch.ServerSocketChannelImpl.socket(ServerSocketChannelImpl.java:128)
        at java.base/sun.nio.ch.PipeImpl$Initializer$LoopbackConnector.run(PipeImpl.java:121)
        at java.base/sun.nio.ch.PipeImpl$Initializer.run(PipeImpl.java:76)
        at java.base/sun.nio.ch.PipeImpl$Initializer.run(PipeImpl.java:61)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
        at java.base/sun.nio.ch.PipeImpl.<init>(PipeImpl.java:171)
        at java.base/sun.nio.ch.SelectorProviderImpl.openPipe(SelectorProviderImpl.java:50)
        at java.base/java.nio.channels.Pipe.open(Pipe.java:155)
        at java.base/sun.nio.ch.WindowsSelectorImpl.<init>(WindowsSelectorImpl.java:142)
        at java.base/sun.nio.ch.WindowsSelectorProvider.openSelector(WindowsSelectorProvider.java:44)
        at java.base/java.nio.channels.Selector.open(Selector.java:295)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:160)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:214)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:227)
        at org.apache.kafka.common.network.Selector.<init>(Selector.java:231)
        at org.apache.kafka.clients.admin.KafkaAdminClient.createInternal(KafkaAdminClient.java:427)
        at org.apache.kafka.clients.admin.Admin.create(Admin.java:69)
        at org.apache.kafka.clients.admin.AdminClient.create(AdminClient.java:49)
        at org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner.createAdminClient(KafkaTopicProvisioner.java:239)
        at org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner.doProvisionConsumerDestination(KafkaTopicProvisioner.java:209)
        at org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner.provisionConsumerDestination(KafkaTopicProvisioner.java:181)
        at org.springframework.cloud.stream.binder.kafka.streams.KafkaStreamsBinderUtils.prepareConsumerBinding(KafkaStreamsBinderUtils.java:84)
        at org.springframework.cloud.stream.binder.kafka.streams.KStreamBinder.doBindConsumer(KStreamBinder.java:102)
        at org.springframework.cloud.stream.binder.kafka.streams.KStreamBinder.doBindConsumer(KStreamBinder.java:52)
        at org.springframework.cloud.stream.binder.AbstractBinder.bindConsumer(AbstractBinder.java:143)
        at org.springframework.cloud.stream.binding.BindingService.doBindConsumer(BindingService.java:169)
        at org.springframework.cloud.stream.binding.BindingService.bindConsumer(BindingService.java:115)
        at org.springframework.cloud.stream.binding.AbstractBindableProxyFactory.createAndBindInputs(AbstractBindableProxyFactory.java:112)
        at org.springframework.cloud.stream.binding.InputBindingLifecycle.doStartWithBindable(InputBindingLifecycle.java:58)
        at java.base/java.util.LinkedHashMap$LinkedValues.forEach(LinkedHashMap.java:647)
        at org.springframework.cloud.stream.binding.AbstractBindingLifecycle.start(AbstractBindingLifecycle.java:57)
        at org.springframework.cloud.stream.binding.InputBindingLifecycle.start(InputBindingLifecycle.java:34)
        at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:182)
        at org.springframework.context.support.DefaultLifecycleProcessor.access$200(DefaultLifecycleProcessor.java:53)
        at org.springframework.context.support.DefaultLifecycleProcessor$LifecycleGroup.start(DefaultLifecycleProcessor.java:360)
        at org.springframework.context.support.DefaultLifecycleProcessor.startBeans(DefaultLifecycleProcessor.java:158)
        at org.springframework.context.support.DefaultLifecycleProcessor.onRefresh(DefaultLifecycleProcessor.java:122)
        at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:894)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:553)
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:758)
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:750)
        at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
        at com.acme.receiver.ApplicationKt.main(Application.kt:88)
...
```

**Comment by bsideup** [comment]
Hi @juergenzimmermann,

Thanks for reporting! The issue comes from our dependency, ByteBuddy, which was updated in master (see #93) but was not released yet.
Could you please try the latest SNAPSHOT (`1.0.3.BUILD-SNAPSHOT`)?

**Comment by juergenzimmermann** [comment]
@bsideup Confirmed: when I use `1.0.3.BUILD-SNAPSHOT` the issue is gone. Hopefully `1.0.3.RELEASE` is released soon because Java 14 is available now.

**Comment by juergenzimmermann** [comment]
@bsideup After upgrading to the newly released `1.0.3.RELEASE` the issue is gone. I'm closing the issue.

**Comment by bsideup** [comment]
Thanks for checking! 👍
